plugins/org.eclipse.thym.ui.importer/src/org/eclipse/thym/ui/importer/CordovaProjectConfigurator.java

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+ * Copyright (c) 2014, 2015 Red Hat, Inc. 
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * 	Contributors:
+ * 	- Mickael Istria (Red Hat Inc.) - initial implementation
+ *******************************************************************************/
+package org.eclipse.thym.ui.importer;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+import org.eclipse.core.resources.IContainer;
+import org.eclipse.core.resources.IFolder;
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IProjectDescription;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Status;
+import org.eclipse.jface.wizard.IWizard;
+import org.eclipse.thym.core.natures.HybridAppNature;
+import org.eclipse.thym.core.platform.PlatformConstants;
+import org.eclipse.ui.wizards.datatransfer.ProjectConfigurator;
+import org.eclipse.wst.jsdt.core.JavaScriptCore;
+
+public class CordovaProjectConfigurator implements ProjectConfigurator {
+
+
+	@Override
+	public boolean shouldBeAnEclipseProject(IContainer container, IProgressMonitor monitor) {
+		return false;
+	}
+
+	@Override
+	public boolean canConfigure(IProject project, Set<IPath> ignoredDirectories, IProgressMonitor monitor) {
+		return project.getFile(PlatformConstants.FILE_XML_CONFIG).exists() || project.getFolder(PlatformConstants.DIR_WWW).getFile(PlatformConstants.FILE_XML_CONFIG).exists();
+	}
```

Comment:
I'll copy paste content in a 1st time, but I think it would be good to export the CanConvertToHybridTester class or to move its canConvert to HybridProjectCreator.